server/src/services/content.ts

```diff
@@ -8,3 +8,7 @@ export const getPostInformation = (id: string): PostInformation => {
   const { content, ...information } = post;
   return information;
 };
+
+export const getPostsInformation = (): string[] => {
+  return getAll();
+};
```

Comment:
Given the previous comment, this can be renamed to  `getPostsIds` and return `Object.keys(getAll())`